automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceNotebooksPage.scala

```diff
@@ -11,18 +11,28 @@ import org.scalatest.selenium.Page
 class WorkspaceNotebooksPage(namespace: String, name: String)(implicit webDriver: WebDriver)
   extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceNotebooksPage] {
 
-  override def awaitReady(): Unit = await enabled sparkClustersHeader
+  override def awaitReady(): Unit = {
+
+    clustersTable.awaitReady()
+  }
+
+  def awaitReadyUnauthorized(): Unit = {
+    find(clusterErrorMessage).isDefined
+  }
```

Comment:
This doesn't actually wait. Should this be `await visible clusterErrorMessage`?

Reply:
yes, I can change that